### DISCUSSION TITLE
How to disable auto date/time in Tabular Editor?

### DISCUSSION THREAD

**Discussion by hayescode** [TabularEditor/TabularEditor3#812] [🙋♀️TE3 Q&A] [answered]
We have converted data models developed in Power BI Desktop and some still have auto date/time enabled. Is there a way to disable auto date/time in Tabular Editor?

**Comment by mlonsk** [comment, marked as answer]
There isn't, that I am aware, any setting in Tabular Editor regarding enabling/disabling auto date/time as this is a Power BI only feature, and not part of the TOM. 
I would think you can just delete the auto date/time tables in your dataset manually and this should take care of the issue. I haven't tested it out, so make sure to take a backup or do it on a copy before deleting in your dataset
